workbench-core/environments/src/postDeployment/cognitoSetup.test.ts

```diff
@@ -47,41 +37,18 @@ describe('CognitoSetup', () => {
       });
     }
 
-    test('run: Create new groups, create new user, add user to Admin group', async () => {
+    test('run: Create new user', async () => {
```

Comment:
Can name this something like 'adminCreateUser is called during admin creation'? Only suggesting that because the test explicitly is checking if the method is called and whether the `run` method throws an error or not; it doesn't actually check if a user was created.